src/main/java/ogc/rs/processes/collectionAppending/CollectionAppendingProcess.java

```diff
@@ -0,0 +1,412 @@
+        cmdLine.addArgument("--config");
+        cmdLine.addArgument("PG_USE_COPY");
+        cmdLine.addArgument("YES");
+        cmdLine.addArgument("-f");
+        cmdLine.addArgument("PostgreSQL");
+        cmdLine.addArgument(
+                String.format("PG:host=%s dbname=%s user=%s port=%d password=%s schemas=public", databaseHost,
+                        databaseName, databaseUser, databasePort, databasePassword), false);
+
+        cmdLine.addArgument("-progress");
+        cmdLine.addArgument("--debug");
+        cmdLine.addArgument("ON");
+
+        cmdLine.addArgument("--config");
+        cmdLine.addArgument("AWS_S3_ENDPOINT");
+        cmdLine.addArgument(awsEndPoint);
+        cmdLine.addArgument("--config");
+        cmdLine.addArgument("AWS_ACCESS_KEY_ID");
+        cmdLine.addArgument(accessKey);
+        cmdLine.addArgument("--config");
+        cmdLine.addArgument("AWS_SECRET_ACCESS_KEY");
+        cmdLine.addArgument(secretKey);
+
+        cmdLine.addArgument(String.format("/vsis3/%s%s", awsBucketUrl, fileName));
+        LOGGER.debug("cmdLine: " + cmdLine);
+        for (String arg : cmdLine.getArguments()) {
+            LOGGER.debug(arg);
+        }
+        return cmdLine;
+    }
+
+    private Future<Void> mergeTempTableToCollection(JsonObject requestInput) {
+
+        Promise<Void> promise = Promise.promise();
+        String jobId = requestInput.getString("jobId");
+        String collectionId = requestInput.getString("collectionsDetailsTableId");
+        String tempTableName = "temp_table_for_" + jobId;
+        LOGGER.debug("temp table name is : " + tempTableName);
+
+        getDbSchema(collectionId)
+                .onSuccess(columns -> {
+                    // Exclude 'id' column
+                    columns.remove("id");
+                    // Every column name should be put into double quotes to avoid the case sensitivity problem of Postgres
+                    String columnNames = columns.stream().map(s -> "\"" + s + "\"").collect(Collectors.joining(", "));
+                    String mergeQuery = String.format(
+                            MERGE_TEMP_TABLE_QUERY,
+                            collectionId,
+                            columnNames,
+                            columnNames,
+                            tempTableName
+                    );
+
+                    pgPool.withConnection(sqlConnection ->
+                            sqlConnection.query(mergeQuery)
+                                    .execute()
+                                    .onSuccess(successHandler -> {
+                                        LOGGER.debug("Merged temp table into main table successfully.");
+                                        promise.complete();
+                                    })
+                                    .onFailure(failureHandler -> {
+                                        LOGGER.error("Failed to merge temp table into main table: " + failureHandler.getMessage());
```

Comment:
Add the job ID to the log message so that we know which one has failed

Reply:
Added the job ID to the log message

https://github.com/datakaveri/ogc-resource-server/pull/139/commits/e09d6b0b9d3029249cd371a0348166c8b7c61fdc